WordPress/metadata/release_notes.txt

```diff
@@ -1,3 +1,6 @@
-- “Get to know your app” flows now stay on screen after rotating your device.
-- Long titles display properly on Stats cards without being cut off.
-- We added the Enter key to Post excerpts to allow for easy line breaks.
+* [***] [internal] Updates the target sdk to 31 - Android 12 [https://github.com/wordpress-mobile/WordPress-Android/pull/17153]
+* [*] Updated About screen to remove mentions of Automattic apps. [https://github.com/wordpress-mobile/WordPress-Android/pull/17282]
```

Comment:
Similar to review comment above, I tried to come up with a dedicated wording that didn't mention the "for WordPress and Jetpack apps to their respective urls" wording here.

From what I saw in your PR (thanks for the screenshots you included there!), while for Jetpack the change is very subtle with very little change in design, for WordPress it seems we removed the "Automattic Family" section with the marbles and mention of other a8c apps (which makes sense given we want to refocus the WP app to .org only), so that's why I came up with this wording for the WP release notes draft.